github-backup.py

```diff
@@ -8,15 +8,30 @@
 """
 
 
-from github import Github
-from argparse import ArgumentParser
+import os
+import errno
+import codecs
+import json
+import itertools
 import subprocess
-import os, os.path
 import logging
+import getpass
+try: #PY3
+    from configparser import SafeConfigParser as ConfigParser
+except ImportError:
+    from ConfigParser import SafeConfigParser as ConfigParser
+from argparse import ArgumentParser
+
+import requests
+import github
 
 LOGGER = logging.getLogger('github-backup')
 
+IsAuthorized = False
+CONFFILE = os.path.join(os.getenv('HOME'), '.github-backup.conf')
+
 def main():
+    global IsAuthorized
     logging.basicConfig(level=logging.INFO)
```

Comment:
Maybe add a space between logger prefix and message?: `format= '[%(levelname)s:%(name)s: %(message)s'`?

Reply:
I'm not sure how I feel about that change.  If you really want it, perhaps add it in a separate PR/commit.

Reply:
Agreed